packages/business-rules/Dockerfile

```diff
@@ -1,13 +1,28 @@
-FROM node:14-alpine
+# Stage 1/3: Build
```

Comment:
I'm not sure if we need those business-rules and common as docker image anymore cause now we copy directly from the source when building the other images